### DISCUSSION TITLE
[RFC] Graph output made interactive

### DISCUSSION THREAD

**Discussion by IPWright83** [vercel/turborepo#7267] [Ideas]
### Describe the feature you'd like to request

I'd like the generated graph from running the [graph](https://turbo-site-git-matt-feature-branch-built-monorepo-handbook.vercel.sh/docs/reference/command-line-reference#--graph) to be understandable when there is a very large graph. At the moment you can only zoom into the top left corner. This example is unusable:

![image](https://user-images.githubusercontent.com/1374775/191208743-2d16cb3e-6aaf-49a4-b0b7-c70dc39c2c00.png)

### Describe the solution you'd like

It would be great if the command to produce the graph spat out something interactive that I can at least pan around and zoom into. 

### Describe alternatives you've considered

You could generate an image too, but I don't think this would be as useful.

**Comment by IPWright83** [comment]
I could potentially find some time to work on the webpage side of this - would be happy to produce a force directed graph in D3.js for example if that's of interest? But I'd need some guidance on how to hook it up to the Go command.

**Comment by chris-olszewski** [comment]
I think getting a webpage that's dedicated to rendering turbo generated DOT files would be amazing. I think it would make sense to use the flow we use for `turbo login` to open up a browser pointed to the webpage with the DOT file loaded. I would be happy to help on the Go side.

**Comment by IPWright83** [comment]
@chris-olszewski I was thinking of just replacing the HTML output to include something with some JavaScript support? I don't think it'd be prudent to modify the existing output formats.
